src/flyout_cursor.ts

```diff
@@ -17,11 +17,28 @@ import * as Blockly from 'blockly/core';
  * This cursor only allows a user to go to the previous or next stack.
  */
 export class FlyoutCursor extends Blockly.Cursor {
+  private flyout: Blockly.IFlyout;
   /**
    * The constructor for the FlyoutCursor.
    */
-  constructor() {
+  constructor(private readonly workspace: Blockly.WorkspaceSvg | any) {
```

Comment:
Where is this passed into the constructor? I don't see that you updated the constructor call, so either this doesn't work or you don't actually need to pass in the workspace.